arch/renesas/src/rx65n/rx65n_usbdev.c

```diff
@@ -2889,7 +2889,7 @@ static int rx65n_epconfigure(FAR struct usbdev_ep_s *ep,
   if (!ep || !desc)
     {
       usbtrace(TRACE_DEVERROR(RX65N_TRACEERR_INVALIDPARMS), 0);
-      printf("ERROR: ep=%p desc=%p\n");
+      printf("ERROR: ep=%p desc=%p\n", ep, desc);
```

Comment:
but it's wrong to call printf in side the code. Please create a new PR to change all printf to the macro defined in debug.h.